source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required (VERSION 3.5)
-set(CMAKE_OSX_SYSROOT iphonesimulator11.2)
+set(CMAKE_OSX_SYSROOT iphonesimulator11.3)
```

Comment:
>3 [](start = 40, length = 1)

Is this change related to the PR?

Reply:
yes, this updates the vesrion

---
In reply to: [188022357](https://github.com/Microsoft/AdaptiveCards/pull/1444#discussion_r188022357) [](ancestors = 188022357)